### ISSUE TITLE
Develop a Rest API

### ISSUE THREAD

**Issue by facine** [siwapp/siwapp-sf3#2]
Would it be possible to implement an API for remote creation: customers, invoices and estimates?

If someone plans a roadmap would love to collaborate.

Thanks!

**Comment by ParisLiakos** [comment]
Yes, i am very interested to it, but havent find the time to implement it. Definitely on the roadmap but needs some research initially.

**Comment by carlosescri** [comment]
We are going to develop an API for the RoR version. If you want you can wait and port it to sf, so they are compatible.

**Comment by ParisLiakos** [comment]
That would actually be awesome, yes..That way the same client can be used.
Definitely a very good suggestion!

**Comment by carlosescri** [comment]
Just to connect tasks, I've created [an issue](siwapp/siwapp#123).

**Comment by JMCA2** [comment]
It would be useful if the Rest API included as feature marking invoices as paid. It would allow to automatically mark invoices as paid when a customer makes the payment through a gateway.
